src/cdk-experimental/menu/menu.spec.ts

```diff
@@ -68,22 +68,124 @@ describe('Menu', () => {
       expect(spy).toHaveBeenCalledWith(menuItems[0]);
     });
   });
+
+  describe('with inner group', () => {
+    let fixture: ComponentFixture<MenuWithNestedGroup>;
+    let menuItems: CdkMenuItemCheckbox[];
+    let menu: CdkMenu;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [MenuWithNestedGroup],
+      }).compileComponents();
+
+      fixture = TestBed.createComponent(MenuWithNestedGroup);
+      fixture.detectChanges();
+
+      menu = fixture.debugElement.query(By.directive(CdkMenu)).injector.get(CdkMenu);
+
+      menuItems = fixture.debugElement
+        .queryAll(By.directive(CdkMenuItemCheckbox))
+        .map(element => element.injector.get(CdkMenuItemCheckbox));
+    }));
+
+    it('should not emit change from root menu ', () => {
+      const spy = jasmine.createSpy('changeSpy for root menu');
+      menu.change.subscribe(spy);
+
+      menuItems.forEach(menuItem => menuItem.trigger());
+
+      expect(spy).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('with inner group render delayed', () => {
+    let fixture: ComponentFixture<MenuWithConditionalGroup>;
+    let menuItems: CdkMenuItemCheckbox[];
+    let menu: CdkMenu;
+
+    const getMenuItems = () => {
+      return fixture.debugElement
+        .queryAll(By.directive(CdkMenuItemCheckbox))
+        .map(element => element.injector.get(CdkMenuItemCheckbox));
+    };
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [MenuWithConditionalGroup],
+      }).compileComponents();
+
+      fixture = TestBed.createComponent(MenuWithConditionalGroup);
+
+      fixture.detectChanges();
+
+      menu = fixture.debugElement.query(By.directive(CdkMenu)).injector.get(CdkMenu);
+
+      menuItems = getMenuItems();
```

Comment:
```suggestion
      fixture = TestBed.createComponent(MenuWithConditionalGroup);
      fixture.detectChanges();
      menu = fixture.debugElement.query(By.directive(CdkMenu)).injector.get(CdkMenu);
      menuItems = getMenuItems();
```
Maybe a _bit_ overboard with the empty lines here